.github/workflows/docs.yml

```diff
@@ -16,46 +16,15 @@ jobs:
   docs:
     name: Build documentation
     runs-on: ubuntu-latest
+    container: registry.ci.openshift.org/coreos/fcos-buildroot:testing-devel
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
       # This is taken from ci/travis-install.sh but should probably be
       # refactored.
       - name: Install dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y \
-            attr \
-            bison \
-            cpio \
-            debhelper \
-            dh-autoreconf \
-            dh-systemd \
-            docbook-xml \
-            docbook-xsl \
-            e2fslibs-dev \
-            elfutils \
-            fuse \
-            gjs \
-            gnome-desktop-testing \
-            gobject-introspection \
-            gtk-doc-tools \
-            libarchive-dev \
-            libattr1-dev \
-            libcap-dev \
-            libfuse-dev \
-            libgirepository1.0-dev \
-            libglib2.0-dev \
-            libgpgme11-dev \
-            liblzma-dev \
-            libmount-dev \
-            libselinux1-dev \
-            libsoup2.4-dev \
-            libcurl4-openssl-dev \
-            procps \
-            zlib1g-dev \
-            python3-yaml
+        run: ./ci/installdeps.sh
```

Comment:
We should probably swallow those up into the buildroot image too. And similarly for rpm-ostree's `ci/install-test-deps.sh`.